cli/command/manifest/create_list.go

```diff
@@ -65,8 +65,13 @@ func createManifestList(dockerCli command.Cli, args []string, opts annotateOpts)
 	manifests := args[1:]
 	logrus.Debugf("retrieving digests of images...")
 	for _, manifestRef := range manifests {
+		namedRef, err := reference.ParseNormalizedNamed(manifestRef)
+		if err != nil {
+			// TODO: wrap error?
+			return err
+		}
 
-		mfstData, _, err := getImageData(dockerCli, manifestRef, targetRef.String(), false)
+		mfstData, _, err := getImageData(dockerCli, namedRef, targetRef.String(), false)
```

Comment:
somewhere in here we need to make sure the namedRef has a tag if one wasn't given by the user (as long as it was passed by digest). as-is, this won't work:  `./build/docker-linux-amd64 -D manifest create clnperez/hello-world hello-world armhf/hello-world s390x/hello-world ppc64le/hello-world`